### ISSUE TITLE
ModuleNotFoundError: No module named 'kraken.lib'; 'kraken' is not a package

### ISSUE THREAD

**Issue by demongolem** [mittagessen/kraken#167]
When installing from pip, that is pip install kraken, I get the falling when running python3 kraken.py.  If I open up a python3 window and type import kraken the same happens.

**Comment by mittagessen** [comment]
* Greg Werner :: 2019-12-23 15:33 Mon:
> When installing from pip, that is pip install kraken, I get the
> falling when running python3 kraken.py.  If I open up a python3 window
> and type import kraken the same happens.

pip installs kraken over an entry point so just running `kraken` from
any directory should work. If `import kraken` in a shell fails something
else is broken. Does the output of `pip list` show an entry for kraken?

**Comment by demongolem** [comment]
In between your reply, I noticed `pip install kraken` give the above results and `pip install .` does allow import from a python shell.  And then yes, pip list shows kraken.  So first of all something different about the pypi version which would cause the reported Issue #167.

On an Ubuntu 18 Virtual machine, yes I can run `kraken` by itself.  On the Mac OS I was trying to originally run on, I do not have admin rights, so I was trying to figure out how to get everything configured.  On Mac OS, I cannot run `kraken` because it is not available from the terminal, only access it through the python shell with import kraken and calling everything directly.

**Comment by mittagessen** [comment]
> In between your reply, I noticed `pip install kraken` give the above
> results and `pip install .` does allow import from a python shell.
> And then yes, pip list shows kraken.  So first of all something
> different about the pypi version which would cause the reported Issue
> #167.

I'm not sure I understand. You say you get an empty package when
installing from pypi?

> On an Ubuntu 18 Virtual machine, yes I can run `kraken` by itself.  On
> the Mac OS I was trying to originally run on, I do not have admin
> rights, so I was trying to figure out how to get everything
> configured.  On Mac OS, I cannot run `kraken` because it is not
> available from the terminal, only access it through the python shell
> with import kraken and calling everything directly.

OK so it's just a simple path issue. On linux .local/bin (where the
binary is linked to when doing a 'local' install without root rights) is
probably in your path but whereever Mac OS pip installs to isn't. If you
run `find . -executable -name 'kraken'` one line should tell you where
it is installed. Amend your $PATH and everything should work as
intended.

**Comment by demongolem** [comment]
Been away a few days for the holiday season.

I have finally located where the local install is so the second part is resolved.  The executable switch did not work for me on Mac OS, but just so for the sake of documentation I wrote `find / -name 'kraken' 2>&1 | grep -v "Permission denied"`

I don't know if that takes care of the first as well, but for my needs anyway, I consider this closed.  Like I said, when working on Mac OS, I simply do not have admin rights and so it is always going to take a little bit more work to do an install like this.
